pkg/applicationserver/io/packages/util_test.go

```diff
@@ -0,0 +1,143 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package packages_test
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"go.thethings.network/lorawan-stack/pkg/component"
+	"go.thethings.network/lorawan-stack/pkg/errors"
+	"go.thethings.network/lorawan-stack/pkg/rpcserver"
+	"go.thethings.network/lorawan-stack/pkg/ttnpb"
+	"go.thethings.network/lorawan-stack/pkg/unique"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+)
+
+func mustHavePeer(ctx context.Context, c *component.Component, role ttnpb.ClusterRole) {
+	for i := 0; i < 20; i++ {
+		time.Sleep(20 * time.Millisecond)
+		if _, err := c.GetPeer(ctx, role, nil); err == nil {
+			return
+		}
+	}
+	panic("could not connect to peer")
+}
+
+type mockIS struct {
+	ttnpb.ApplicationRegistryServer
+	ttnpb.ApplicationAccessServer
+	applications     map[string]*ttnpb.Application
+	applicationAuths map[string][]string
+}
+
+func startMockIS(ctx context.Context) (*mockIS, string) {
```

Comment:
Do we really need another duplicate of this?
Why not just use https://github.com/TheThingsNetwork/lorawan-stack/blob/131cb27619bba1869a32bed4c2f0ba9e604637ad/pkg/util/test/cluster.go#L79 ?

Example usage: https://github.com/TheThingsNetwork/lorawan-stack/blob/131cb27619bba1869a32bed4c2f0ba9e604637ad/pkg/networkserver/networkserver_util_test.go#L361-L365
https://github.com/TheThingsNetwork/lorawan-stack/blob/131cb27619bba1869a32bed4c2f0ba9e604637ad/pkg/networkserver/networkserver_flow_test.go#L74-L76

Reply:
This is happening in a lot of places, and we already have plans to replace it throughout the repo, so I'm okay with saving some time now and changing it all at the same time.

Reply:
References https://github.com/TheThingsNetwork/lorawan-stack/issues/1389